Gopkg.toml

```diff
@@ -27,25 +27,62 @@
 ignored = ["github.com/kubeflow/kubebench"]
 
 [[constraint]]
-  name = "github.com/ghodss/yaml"
-  version = "1.0.0"
+  branch = "master"
+  name = "github.com/lib/pq"
 
 [[constraint]]
-  name = "github.com/kubeflow/pytorch-operator"
-  branch = "master"
+  branch = "v2"
+  name = "gopkg.in/yaml.v2"
 
 [[constraint]]
-  name = "github.com/kubeflow/tf-operator"
-  branch = "master"
+  name = "github.com/argoproj/argo"
+  version = "2.2.0"
 
 [[constraint]]
+  branch = "master"
+  name = "github.com/hashicorp/golang-lru"
+
+[[override]]
   name = "k8s.io/api"
-  branch = "release-1.9"
+  version = "kubernetes-1.12.2"
 
-[[constraint]]
+[[override]]
+  name = "k8s.io/apiextensions-apiserver"
+  version = "kubernetes-1.12.2"
+
+[[override]]
   name = "k8s.io/apimachinery"
-  branch = "release-1.9"
+  version = "kubernetes-1.12.2"
+
+[[override]]
+  name = "github.com/json-iterator/go"
+  revision = "f2b4162afba35581b6d4a50d3b8f34e33c144682"
+
+[[override]]
+  name = "github.com/Azure/go-autorest"
+  revision = "1ff28809256a84bb6966640ff3d0371af82ccba4"
+
+[[override]]
+  name = "k8s.io/client-go"
+  version = "kubernetes-1.12.2"
+
+[[constraint]]
+  name = "github.com/kubeflow/pytorch-operator"
+  branch = "master"
+
+[[constraint]]
+  name = "github.com/kubeflow/tf-operator"
+  version = "v0.2.0-rc1"
+
+[[override]]
+  name = "k8s.io/code-generator"
+  branch = "master"
+
+[[override]]
+  name = "k8s.io/kube-openapi"
+  branch = "release-1.10"
 
 [prune]
   go-tests = true
-  unused-packages = true
+#  unused-packages = true
```

Comment:
set for argo pkg only, do not set globally. you may refer to https://golang.github.io/dep/docs/Gopkg.toml.html#prune.